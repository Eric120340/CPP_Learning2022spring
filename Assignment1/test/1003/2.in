1 8
82 39 25 26 52 6 39 17 
8 5
52 84 58 61 77 
97 68 25 96 98 
57 36 32 95 39 
21 96 82 64 88 
12 23 72 46 61 
48 71 65 53 62 
33 56 97 90 68 
74 86 35 98 81 
