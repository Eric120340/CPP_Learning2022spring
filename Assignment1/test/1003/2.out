13475 18204 17175 20671 20972 
